django_comments_xtd/api/frontend.py

```diff
@@ -25,7 +25,7 @@ class CommentBoxDriver(object):
     def get_queryset(cls, ctype, obj, request):
         return XtdComment.objects.filter(
             content_type=ctype,
-            object_pk=obj.pk,
+            object_pk=obj._get_pk_val(),
```

Comment:
I'm not sure why this change is needed. The `pk` attribute is defined [here](https://github.com/django/django/blob/3e3e8141662c70d74d3aaef9d4b053c9636ea0d7/django/db/models/base.py#L680) as a property that resolves to `_get_pk_val` for the getter and to `_set_pk_val` for the setter. So using `obj.pk` calls `obj._get_pk_val()`. What am I missing?

Reply:
I am not sure. Probably it was needed when I was exchanging the function in my hacky implementation of Proxy models.
But also might not be needed even then and I changed it because I taught it was needed.

I will change it to pure `pk`s and will see if everything works.